services/manager/lib/io/http/index.js

```diff
@@ -1,9 +1,18 @@
 const express = require('express');
 const ip = require('ip');
 const pathLib = require('path');
+const serveIndex = require('serve-index')
 
 const logger = require('../logger');
 
+const directoryListing = ({ fileList, directory }, callback) => {
+  const listing = fileList
+    .filter((f) => (f['name'][0] != '.'))
```

Comment:
The docs for `serve-index` makes it sound like it [shouldn't be listing hidden files](https://www.npmjs.com/package/serve-index#hidden). Is that not happening?

Reply:
It includes the `..` path up to the parent directory